core/platform/image/gae_image_services.py

```diff
@@ -79,9 +80,36 @@ def compress_image(image_content, scaling_factor):
                     MAX_RESIZE_DIMENSION_PX, float(max(width, height))))
             new_width = int(width * new_scaling_factor)
             new_height = int(height * new_scaling_factor)
-        return images.resize(
-            image_data=image_content,
-            width=min(new_width, MAX_RESIZE_DIMENSION_PX),
-            height=min(new_height, MAX_RESIZE_DIMENSION_PX))
+            new_image_dimensions = (
+                min(new_width, MAX_RESIZE_DIMENSION_PX),
+                min(new_height, MAX_RESIZE_DIMENSION_PX))
+
+            # Thumbnail doesn't work for enlarging images.
```

Comment:
I think you can drop this, because no one will know what "Thumbnail" means in this context. But if you want to keep it, you can say "The image.thumbnail() command doesn't work..." so that people know what you're talking about.

Reply:
Done